udata_front/theme/gouvfr/templates/dataset/search-result.html

```diff
@@ -3,7 +3,7 @@
 <article class="dataset-card dataset-search-result py-xs">
     {% if dataset.organization %}
     <div class="card-logo">
-        <img alt=" {{ dataset.organization.title }}"
+        <img alt="{{ dataset.organization.name|placeholder_alt(dataset.organization.logo) }}"
```

Comment:
Somehow in the audit, they wanted an empty alt here (P8), but I'm not convinced by the logic.

Reply:
Before the fix, there was a ` ` (space character) before the organization `title` so the attribute was always `alt=" "` on the audited page. Now, the space is gone and we use `name` instead.
My understanding is that the reporter treated this case because there was an alt with a space and couldn't know what was supposed to be here. I tried to stay consistent with P4 about search results.
What do you think?

Reply:
Ah yes, that makes sense, indeed the trailing space was probably the reason they asked to set an empty alt. I think your reasoning makes sense.